src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  &__image-imac {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 36px;
+  }
+
+  &__product-description {
+    display: block;
+    padding: 0 16px 16px;
+  }
+
+  &__description {
+    width: 100%;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 0;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    padding: 4px 0 16px;
+    margin: 0;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: flex-end;
+    width: 53px;
+    height: 16px;
```

Comment:
```suggestion

```
The size should depend on the content. It's bad practice to hardcode these values